kokoro/builds/gcc9_relwithdebinfo/build.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 The Orbit Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# Fail on any error.
+set -e
+
+readonly DIR="/mnt/github/orbitprofiler"
+readonly SCRIPT="${DIR}/kokoro/builds/gcc9_relwithdebinfo/build.sh"
+readonly CONAN_PROFILE="gcc9_relwithdebinfo"
+
+if [ "$0" == "$SCRIPT" ]; then
+  # We are inside the docker container
+
+  pip3 install conan==1.27.1 conan-package-tools==0.34.0
+
+  echo "Installing conan configuration (profiles, settings, etc.)..."
+  ${DIR}/third_party/conan/configs/install.sh || exit $?
+
+  CRASHDUMP_SERVER=""
+
+  # Building Orbit
+  mkdir -p "${DIR}/build/"
+  cp -v "${DIR}/third_party/conan/lockfiles/linux/${CONAN_PROFILE}/conan.lock" \
+        "${DIR}/build/conan.lock"
+  sed -i -e "s|crashdump_server=|crashdump_server=$CRASHDUMP_SERVER|" \
+            "${DIR}/build/conan.lock"
+  conan install -u -pr ${CONAN_PROFILE} -if "${DIR}/build/" \
+          --build outdated \
+          -o crashdump_server="$CRASHDUMP_SERVER" \
```

Comment:
Do we need to define `CRASHDUMP_SERVER=""` and pass it here?

Reply:
No, we don't need that, but I didn't wanted to change the code too much compared to the build-script from the `gcp_ubuntu`-job, since my intention is to eventually merge them back together into one build script. Does that make sense?

Reply:
Yes, sure. Thanks for the clarification